content/en/docs/best-practices/general-principles/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: General Principles
+description: The general principles for your Istio deployments.
+weight: 1
+icon: best-practices
+---
+
+We've identified the following general principles to help you get the most out
+of your Istio deployments.
+
+## Make your deployment simple
+
+When choosing a [deployment model](/docs/concepts/deployment-models),
+**prefer simplicity over complexity** wherever possible.
+
+## Make your deployment fast
```

Comment:
This feels weird as a statement. It isn't really making your deployment fast, which sounds like something about getting a good CI/CD pipeline in place.

This is about favoring geographic proximity to users, maybe we can say something about deploying close to your users?

BTW I get the nice "Make your deployment X" but I don't think it is works as well as having each section be more explicit:
- Prefer simpler deployment models
- Place clusters near your users
- Use multiple availability zones

Reply:
+1 to what Sven said. Saying "Make your deployment fast" as a best practice is a bit redundant. It's not like operator are going to say "Gosh, I wish I would have thought of that, I was trying to make my deployment slow" :-)

Instead, provide a list of specific actions and for each explain the benefits.